tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -654,7 +654,10 @@
             "type": "object",
             "description": "An set of key-value pairs specifying tags describing the pipeline. The tag names are open, they might be more constrained in the future.",
             "additionalProperties": {
-                "type": "string",
+                "type": ["string", "array"],
+                "items": {
+                    "type": "string"
+                },
```

Comment:
This is not backward compatible, which we try to avoid whenever possible. I think it would make sense that a tag can be a string or an array, so that users can still specify just a string when they don't have multiple values. Could you update the schema and the tests accordingly?

Reply:
It already allows for both. `"type": ["string", "array"]` means that the value can be either a string or an array. Also, the `"items"` restriction will only be applied if the value is an array and will be ignored if it's a string. It's explained here: https://cswr.github.io/JsonSchema/spec/multiple_types/

Reply:
Sorry I had missed that, that's perfect!